src/g/StageScale.js

```diff
@@ -24,18 +24,46 @@ module.exports = StageScale = Model.extend({
       { columnWidth: 15, markerStepSize: 2, stepSize: 1 },
       { columnWidth: 20, markerStepSize: 1, stepSize: 1 },
       { columnWidth: 30, markerStepSize: 1, stepSize: 1 },
+      { columnWidth: 45, markerStepSize: 1, stepSize: 1 },
     ],
   },
 
   initialize: function(args) {
+    const categories = this.get('scaleCategories');
+    const initialColumnWidth = this.g.zoomer.get('columnWidth') || this._getScaleInfo().columnWidth;
+
+    /* if the global columnWidth setting doesn't match any of our categories
+     * then create a category that does match and add it to a sensible place
+     * in the list
+     */
+    var category = _.find( categories, function(c) { return c.columnWidth == initialColumnWidth });
```

Comment:
Another FYI - you can directly search for a key Value pair  with `{columnWidth: initialColumnWidth}`